pages/common/npm.md

```diff
@@ -1,10 +1,7 @@
 # npm
 
-> Node package manager, to manage Node.js projects and install module dependencies.
-
-- Create a new project in the current folder:
-
-`npm init`
+> JavaScript and Node.js package manager.
+> Manage Node.js projects and their module dependencies.
 
 - Download all dependencies referenced in package.json:
```

Comment:
In reality you rarely do `npm init`. But you often do `npm install`. Honestly if we decide to remove any example, then `npm init` should be 1st candidate. All it does, it just helps you creating package.json. In many cases you just do it yourself, or clone repo, or sometimes even don't use package.json at all, as it was on tldr some.

In contrast, `npm install` is the most common, the most popular command. That's why `npm` is created - to do `npm install`.